src/pages/documentation/platform-assurance-sdk/tabs/index.md

```diff
@@ -199,7 +194,14 @@ When using React Native, register AEP Assurance with Mobile Core in native code
 
 When using Flutter, register AEP Assurance with Mobile Core in native code as shown on the Android and iOS tabs. --->
 
-<Variant platform="ios" task="implement-assurance" repeat="12"/>
+
+<Variant platform="android" task="implement-assurance" repeat="1"/>
+
+Deep link is the recommended way of connecting to an Assurance session when using the Android SDK. Assurance SDK on Android is already setup to handle incoming intents to your app. It is sufficient to [add an intent filter for incoming links in your app](https://developer.android.com/training/app-links/deep-linking) to complete the deep link configuration. The combination of `android:host` and `android:scheme` (in the form of `<host>://<scheme>`) of this intent filter will serve as the Base URL while creating a session in the [Adobe Experience Platform Assurance UI](https://experience.adobe.com/assurance)
```

Comment:
"Deep linking is the best way to connect to an Assurance session when..."

"You can [add an intent filter for incoming links in your app] to complete the..."

"The combination of... for this intent filter will serve..."